src/pages/NewUser/NewUser.tsx

```diff
@@ -22,7 +22,7 @@ const NewUser: FC = () => {
 
 	return (
 		<UniversalModal
-			title={'Create New User nahui'}
+			title={''}
```

Comment:
```suggestion
			title={'New user'}
```

Reply:
Here should be an empty string.
This modal opens form which already have description 'New user'
![image](https://github.com/user-attachments/assets/5fa8d8a2-c8a3-4b5e-ae38-cd2e055035cc)

Reply:
Don't you think that that title should be set here for more clarity? Because now it looks like there should be a modal without a title, when the title exists. Maybe I'm confusing something, but when I use modal I need to control the title in the same place where the modal is called.

Reply:
i`ve deleted "New user" string/title from Form props, and added this title to modal props
Now it`s apear little in other plase in modal, and i hawe control in one place